### ISSUE TITLE
FloatingActionButtonShadowNode compilation error

### ISSUE THREAD

**Issue by fdnhkj** [cesardeazevedo/react-native-bottom-sheet-behavior#4]
Hello!

Thanks for your repo!
I am trying to use it but I am facing following issue when compiling.

> :react-native-bottom-sheet-behavior:compileReleaseJavaWithJavac
> /Users/user/vscode-projects/reactNativeProject/node_modules/react-native-bottom-sheet-behavior/android/src/main/java/com/bottomsheetbehavior/FloatingActionButtonShadowNode.java:10: error: FloatingActionButtonShadowNode is not abstract and does not override abstract method measure(CSSNodeAPI,float,CSSMeasureMode,float,CSSMeasureMode,MeasureOutput) in MeasureFunction
> public class FloatingActionButtonShadowNode extends LayoutShadowNode implements CSSNode.MeasureFunction {
>        ^
> /Users/user/vscode-projects/reactNativeProject/node_modules/react-native-bottom-sheet-behavior/android/src/main/java/com/bottomsheetbehavior/FloatingActionButtonShadowNode.java:19: error: method does not override or implement a method from a supertype
>     @Override
>     ^
> Note: /Users/user/vscode-projects/reactNativeProject/node_modules/react-native-bottom-sheet-behavior/android/src/main/java/com/bottomsheetbehavior/ResourceDrawableIdHelper.java uses or overrides a deprecated API.
> Note: Recompile with -Xlint:deprecation for details.
> Note: /Users/user/vscode-projects/reactNativeProject/node_modules/react-native-bottom-sheet-behavior/android/src/main/java/com/bottomsheetbehavior/FloatingActionButtonManager.java uses unchecked or unsafe operations.
> Note: Recompile with -Xlint:unchecked for details.
> 2 errors
> :react-native-bottom-sheet-behavior:compileReleaseJavaWithJavac FAILED
> 
> FAILURE: Build failed with an exception.
> 
> What went wrong:
> Execution failed for task ':react-native-bottom-sheet-behavior:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

Here are the dependencies I use

``` json
  "dependencies": {
    "react": "15.3.1",
    "react-native": "^0.32.0",
    "react-native-bottom-sheet-behavior": "0.0.4",
    "react-native-maps": "0.8.2",
    "react-native-router-flux": "^3.35.0",
    "react-redux": "^4.4.5",
    "redux": "^3.6.0",
    "redux-thunk": "^2.1.0"
  },
```

**Comment by cesardeazevedo** [comment]
Hi, thanks for report, sorry the delay, can you post your`android/build.gradle`?, i remember something related to it on changing`url "$projectDir/../../node_modules/react-native/android"` to `url "$rootDir/../node_modules/react-native/android"`, but i can be wrong...

**Comment by cesardeazevedo** [comment]
Nevermind, there's something wrong on react-native android dependency, i'm trying to figure it out.

**Comment by cesardeazevedo** [comment]
I've fixed on 0.0.5, let me know if you got it.

**Comment by Andreyco** [comment]
Having very same issue.
RN 0.38.0
React 15.4.1

**Comment by Andreyco** [comment]
Downloaded example (runs on 0.36 - has some issues with running, but compiles, that's okay).
- upgraded to 0.37 (JS issues with running out of the box, but compiles)
- upgraded to 0.38, does not compile

**Comment by cesardeazevedo** [comment]
I think they did some break changes on the CSS-Layout on version 0.38, i will fix it soon.

Can you tell me what issues you have when running with 0.36?

**Comment by Andreyco** [comment]
Don't remember exact message but it's related to root "BSBExample" component not being registered.
It's not eating this...

```
AppRegistry.registerComponent('BSBExample', () => BSBExample)
```

Going to look into this and move example up to 0.37 and possibly to 0.38 when you push fix.

**Comment by Andreyco** [comment]
- **0.36 + example:** works as expected, false alarm here
- **0.37:** won't work, unless you drop `react-native-vector-icons`. RN 0.37 introduces issue related to `Animated` library -> https://github.com/oblador/react-native-vector-icons/releases
- **0.38**: fixes issue introduced in 0.37, but breaks your code, lol -> https://github.com/exponentjs/react-native/commit/ae5887ab4757cdc0120000ba189d0131eafddb79

Going straight to 0.38 and fixing your code is way to go, imho.
